src/core/index.js

```diff
@@ -1,11 +1,13 @@
 import { Session } from "./session"
+import { Cache } from "./cache"
 import { PageRenderer } from "./drive/page_renderer"
 import { PageSnapshot } from "./drive/page_snapshot"
 import { FrameRenderer } from "./frames/frame_renderer"
 import { FormSubmission } from "./drive/form_submission"
 
 const session = new Session()
-const { cache, navigator } = session
+const cache = new Cache(session)
+const { navigator } = session
```

Comment:
This appears to be reverting some recent changes made to `main`:

```suggestion
const { cache, navigator } = session
```